app/assets/stylesheets/survey_modules/survey__admin-row.styl

```diff
@@ -102,3 +102,57 @@
 
 .surveys__admin-header--question-groups--edit
   margin-bottom 0 !important
+
+// Preview dropdown styles
+.preview-dropdown
```

Comment:
Should these styles be scoped more specifically? `.preview-dropdown` isn't a classname that is specific to `survey__admin-row`, so these should either be in a different file (if they likely to be generally useful in other contexts) or they should follow the naming convention of css classes that are specific to the filename.